solutions/gold/ioi-11-crocodile.mdx

```diff
@@ -0,0 +1,94 @@
+---
+id: ioi-11-crocodile
+source: IOI
+title: 2011 - Crocodile
+author: Justin Ji
+---
+
+[Official Analysis](https://dmoj.ca/problem/ioi11p4/editorial)
+
+<Spoiler title="Hint 1">
+
+Instead of trying to traverse from our starting node to an endpoint, consider
+starting from our endpoints and try to reach our starting node.
+
+</Spoiler>
+
+<Spoiler title="Hint 2">
+
+Obviously, the crocodile will try to prevent us from taking the best path from
```

Comment:
no obviously, even if you and i both think it is

Reply:
oops